docker-compose.dev.yml

```diff
@@ -0,0 +1,101 @@
+version: '3'
```

Comment:
why a dev dockerfile when the main dockerfiles are intended for dev already?

Reply:
Just used an extra file to play around with for a bit. Merged them now.